ExtensionHandler/Linux/src/AzureRM.py

```diff
@@ -107,6 +131,11 @@ def set_error_status_and_error_exit(e, operation_name, code = -1):
   if(len(error_message) > Constants.ERROR_MESSAGE_LENGTH):
     error_message = error_message[:Constants.ERROR_MESSAGE_LENGTH]
   handler_utility.error('Error occured during {0}. {1}'.format(operation_name, error_message))
+  try:
+    event_logger.log_new_event("extension_failed", 'Error occured during {0}. {1}'.format(operation_name, error_message))
+  except Exception as e:
```

Comment:
There's no need to use `except Exception as e:` as you're not doing anything with `e`, thus `except Exception` is enough.

Reply:
Sure, changes made